mne/preprocessing/ica.py

```diff
@@ -206,23 +207,26 @@ class ICA(ContainsMixin):
     ch_names : list-like
         Channel names resulting from initial picking.
         The number of components used for ICA decomposition.
-    ``n_components_`` : int
+    n_components_ : int
         If fit, the actual number of components used for ICA decomposition.
     n_pca_components : int
         See above.
     max_pca_components : int
         The number of components used for PCA dimensionality reduction.
     verbose : bool, str, int, or None
         See above.
-    ``pca_components_`` : ndarray
+    pre_whitener_ : ndarray
+        Matrix used to pre-whiten the data prior to PCA; by default, this
+        scales all channels to unit variance.
+    pca_components_ : ndarray
         If fit, the PCA components
-    ``pca_mean_`` : ndarray
+    pca_mean_ : ndarray
```

Comment:
I really think you should document the shapes using `n_channels` and the other attributes such as `n_components_`

Reply:
Can we do that in #4856?

Reply:
I think it's okay to postpone the shape stuff until we fully tackle the full description of operations, which is indeed covered by #4856 (this one can just fix some minor issues incrementally).

Now that #5120 is merged, can you rebase this one @cbrnr then we can merge?

Reply:
Done.